lib/osdep_linux.c

```diff
@@ -131,7 +131,7 @@ int compare_pm_table(void *buffer, size_t size)
 	return memcmp(buffer, phy_map, size);
 }
 
-bool is_using_smu_driver()
+inline bool is_using_smu_driver()
```

Comment:
Why inline here....

Reply:
it's just an inline request for the compiler. the function is short, I thought that maybe in the future if it is necessary to have it in the same object it would be more efficient.

Reply:
> it's just an inline request for the compiler. the function is short, I thought that maybe in the future if it is necessary to have it in the same object it would be more efficient.

If it’s in the same object compiler will try to inline it as well (as long as the symbol is not preemptible)

If you are going to inline something, use static inline in headers. standalone inline keyword makes no sense at all.

I’d prefer not to have this kind of hack in our code base. It makes no sense to squeeze out a couple of instructions when your CPU is running at multi gigahertz.

Reply:
okay, I understand the reasons. do you prefer that I remove the commit or just remove inline?